etc/disable-interpreters.inc

```diff
@@ -0,0 +1,38 @@
+# This file is overwritten during software install.
```

Comment:
Maybe lua could also be added?

Reply:
Added in b6c3e7681df89f7ea8ea9ac7ece1181f0601e86c